time_tracker_api/time_entries/time_entries_namespace.py

```diff
@@ -46,11 +54,13 @@
         required=True,
         title='Identifier',
         description='The unique identifier',
+        example=faker.uuid4()
     ),
     'running': fields.Boolean(
         readOnly=True,
         title='Is it running?',
-        description='Whether this time entry is currently running or not'
+        description='Whether this time entry is currently running or not',
+        example=faker.boolean()
```

Comment:
Actually, `running` has to be `true` if the `end_date` has no value. I think that `end_date` has to have a value or not.